inc/features/blocks/variant/register.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Experience Block Server Side.
+ *
+ * @phpcs:ignoreFile HM.Files.FunctionFileName.WrongFile
+ *
+ * @package altis-experiments
+ */
+
+namespace Altis\Experiments\Features\Blocks\Variant;
+
+use function Altis\Experiments\Features\Blocks\get_block_settings;
+use function Altis\Experiments\Utils\get_asset_url;
+
+const BLOCK = 'variant';
+
+function setup() {
+	$block_data = get_block_settings( BLOCK );
+
+	// Queue up JS files.
+	add_action( 'enqueue_block_editor_assets', __NAMESPACE__ . '\\enqueue_assets' );
+
+	// Register the block.
+	register_block_type( $block_data['name'], [
+		'attributes' => $block_data['settings']['attributes'],
+		'render_callback' => __NAMESPACE__ . '\\render_block',
+	] );
+}
+
+function enqueue_assets() {
+	wp_enqueue_script(
+		'altis-experiments-features-blocks-variant',
+		get_asset_url( 'features/blocks/variant.js' ),
+		[],
+		null
```

Comment:
Should have version?

Reply:
negatif, uses a content hash in the built file name